### PR TITLE
Split tfjs loading related code to loader.js

### PR DESCRIPTION
**Pull request by axinging**
This change is trying to decouple loading related code from index.html.
With this change, we can also easily implement our own tfjs demo with the latest tfjs build, below is example:

```
    <script src="loader.js"></script>
    <script>
        'use strict';

        async function runTFJS(backend) {
            await tf.setBackend(backend);
            await tf.ready();
            tf.env().set('WEBGPU_CPU_FORWARD', false);
            const a = tf.tensor3d([1, 11, 2, 22, 3, 33, 4, 44], [2, 2, 2]);
            const b = tf.tensor3d([1, 11, 2, 22, 3, 33, 4, 44], [2, 2, 2]);

            const result = tf.add(a, b);
            console.log(await result.data());
        }

        async function loadTFJSDemo() {
            const urlState = new URLSearchParams(location.search);
            let localBuild = [];
            if (urlState && urlState.has('localBuild')) {
                localBuild = urlState.get('localBuild').split(',');
            }
            await loadTFJS(localBuild);
            return urlState.get('backend');
        }

        (async function () {
            const backend = await loadTFJSDemo();
            await runTFJS(backend);
        })();
    </script>
```

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6540)

### CODE REVIEW

**Comment by axinging** [comment]
@qjia7 @haoyunfeix @gyagp PTAL

**Comment by qjia7** [comment]
No preference for me since the original code is only for benchmarks.  You can easily write your own tests without `loader.js`.
Leave it to Ping/Matt to decide.

**Review by mattsoulanille — APPROVED** [review]
LGTM

**Review by pyu10055 — APPROVED** [review]

File: e2e/benchmarks/local-benchmark/loader.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+// Promise is used to guarantee scripts are loaded in order.
+async function loadScript(url) {
```
[the remark this answers is not in the document]
Comment (axinging) [reply e2e/benchmarks/local-benchmark/loader.js]: Yes, done

**Pull request merged**